apis/logging/v1/clusterlogging_types.go

```diff
@@ -54,11 +54,6 @@ type ClusterLoggingSpec struct {
 	//
 	// +nullable
 	Curation *CurationSpec `json:"curation,omitempty"`
-
-	// Specification for Forwarder component for the cluster
-	//
-	// +nullable
-	Forwarder *ForwarderSpec `json:"forwarder,omitempty"`
```

Comment:
We can not remove this but will need to mark it as deprecated for upgrade and backwards compatibility reasons